test/stress/load-test/main.go

```diff
@@ -42,6 +42,8 @@ var (
 	nonceChaos          int
```

Comment:
I kinda want the stress tool to compile to a binary named `stress`.
`test/stress/stats` doesn't really fit.

Maybe
`test/stress` -- stress tool
`test/statsmon` -- stats collection tool.

Or put everything under `test/cmd`: `test/cmd/stress` and `test/cmd/stats`?

Reply:
okay